circle.yml

```diff
@@ -304,13 +304,13 @@ jobs:
   sanitizers-macos:
     executor: macos
     environment:
-      BUILD_TYPE: RelWithDebInfo
-      CMAKE_OPTIONS: -DENABLE_ASSERTIONS=ON -DSANITIZE=address,undefined,nullability,implicit-unsigned-integer-truncation,implicit-signed-integer-truncation
       UBSAN_OPTIONS: halt_on_error=1
```

Comment:
This is not a build, but runtime parameter, hence not move under `build` settings?

Reply:
It must be the environment variable anyway (libubsan checks it), so this way of setting it is fine.